dao/src/main/java/greencity/repository/BagRepository.java

```diff
@@ -58,11 +58,10 @@ public interface BagRepository extends JpaRepository<Bag, Integer> {
      * method, that returns {@link Bag}'s info.
      * 
      * @param orderId order id {@link Long}
-     * @author Nazar Struk
+     * @author José Castellanos
```

Comment:
you should not change `@author`
If you want you can add one more `@author`

Reply:
done